src/openzaak/config/templates/config/config_wizard.html

```diff
@@ -0,0 +1,36 @@
+{% extends "master.html" %}
+{% load i18n %}
+
+{% block content %}
+
+<div class="px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+    <h1 class="display-4">Service configuratie: {{ wizard.steps.current| upper }}</h1>
+    <p class="lead">Step {{ wizard.steps.step1 }} of {{ wizard.steps.count }}</p>
+</div>
+
+<div class="container">
+    {% load i18n %}
```

Comment:
already loaded at the top of the template :)

Reply:
fixed, thanks!